tools/review_pull_request.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+# License AGPLv3 (http://www.gnu.org/licenses/agpl-3.0-standalone.html)
+
+import argparse
+from collections import Counter
+from datetime import datetime, timedelta
+
+from github_login import login, wrap_github_call
+
+from oca_projects import OCA_USERNAME, OCA_REPOSITORY_NAMES
+
+contributions_cache = None
+
+
+def get_contribution_counts():
+    global contributions_cache
+
+    if contributions_cache is not None:
+        return contributions_cache
+
+    github = login()
+    contributions_cache = Counter()
+
+    for repository_name in OCA_REPOSITORY_NAMES:
+        repository = wrap_github_call(
+            github.repository,
+            [OCA_USERNAME, repository_name]
+        )
+        for user in wrap_github_call(repository.contributors):
+            contributions_cache[user.id] += user.contributions_count
+
+    return contributions_cache
+
+
+def tag_new_pull_requests(repository_name, dry_run=False):
+    github = login()
+    repository = wrap_github_call(
+        github.repository,
+        [OCA_USERNAME, repository_name]
+    )
+    contributions = get_contribution_counts()
+
+    for pull_request in wrap_github_call(repository.pull_requests):
+        if contributions[pull_request.user.id] < 3:
```

Comment:
I think this (and all other hardcoded values further down) should be arguments with the curernt value as default value